pkg/config/node_test.go

```diff
@@ -338,6 +343,148 @@ func deleteTempResolvConf() {
 	os.Remove("/tmp/resolvConf")
 }
 
+var (
+	icFilePath    = "/tmp/install-config.yaml"
```

Comment:
Still hardcoded `/tmp` here

Reply:
That was just the initial value that was updated after the file was created. Anyways, updated.